pkg/scheduler/plugins/proportion/resource_division/resource_division.go

```diff
@@ -23,18 +23,18 @@ type remainingRequestedResource struct {
 	remainingAmount float64
 }
 
-func SetResourcesShare(totalResource rs.ResourceQuantities, queues map[common_info.QueueID]*rs.QueueAttributes) {
+func SetResourcesShare(totalResource rs.ResourceQuantities, kValue float64, queues map[common_info.QueueID]*rs.QueueAttributes) {
```

Comment:
I dont see any usage of `kValue` beside passing it around.

Reply:
It will be used in the next PRs, I wanted to make them smaller and with less boilerplate